src/VirtoCommerce.Platform.Caching/PlatformMemoryCache.cs

```diff
@@ -43,12 +46,26 @@ public virtual void Remove(object key)
         }
 
 
-        protected bool CacheEnabled => _cachingOptions.CacheEnabled;
-        protected TimeSpan? AbsoluteExpiration => _cachingOptions.CacheAbsoluteExpiration;
-        protected TimeSpan? SlidingExpiration => _cachingOptions.CacheSlidingExpiration;
+        public virtual bool CacheEnabled
```

Comment:
Why have you changed accessibility for these properties?

Reply:
For extensibility reasons I added these properties to `IPlatformMemoryCache`. Interface members can't be implemented as protected (except implicit implementation aka `void IPlatformMemoryCache.CacheEnabled`). They also should be public for same extensibility reasons - otherwise you can't disable cache without making your own `PlatformMemoryCache` ancestor, which isn't best solution especially because of existing `RedisPlatformMemoryCache`

Reply:
you don't need to extend the entire interface `IPlatformMemoryCache` due to these extends the responsibility and absolutely not required for this task. Just call protected properties in the descendant.

Reply:
`IPlatformMemoryCache` inherit `IMemoryCache` which is responsible for manipulations with cache (get, add, remove cache entries). It also provide cache options for same cases - cache disability & expirations. So it's ok to add them here. I'm just trying to make minor improvements to prevent problems with extensibility later.

But as you wish.

@OlegoO And you saying "make small improvements if you see problems". How?)

Reply:
Well, I returned them back. In addition to extensibility, there is also another reason: there is no other way to determine disabled cache or not. It also making code not testable.

Reply:
Fixed